shacl12-vocabularies/shacl-shacl.ttl

```diff
@@ -137,16 +141,16 @@ shsh:ShapeShape
 	] ;
 	sh:property [
 		sh:path sh:ignoredProperties ;
-		sh:node shsh:ListShape ;        # ignoredProperties-node
-		sh:maxCount 1 ;                 # multiple-parameters
-	] ;
-	sh:property [
-		sh:path ( sh:ignoredProperties [ sh:zeroOrMorePath rdf:rest ] rdf:first ) ;
-		sh:nodeKind sh:IRI ;            # ignoredProperties-members-nodeKind
+		sh:memberShape [ sh:nodeKind sh:IRI ] ; # ignoredProperties-memberShape
+		sh:minListLength 1 ;                    # ignoredProperties-minListLength
```

Comment:
Added in #416 .

Reply:
This is defining a constraint on `sh:ignoredProperties` which is not added in #416. It is not defining the `sh:memberShape` or `sh:minListLength` constraint which is what is done in #416.